docs/installation/systemd-upgrading.md

```
@@ -6,8 +6,44 @@ automatically restart the Control Plane service after the update is complete.
 > The package upgrade will preserve any modifications to the configuration file
 > at `/etc/pgedge-control-plane/config.json`.
 
+We recommend updating the Control Plane using the pgEdge Enterprise package
+repositories you configured in the [Packages](systemd-installation.md#packages)
+section. If you installed the Control Plane manually, see [Updating from
+GitHub Releases](#updating-from-github-releases) below.
+
 ### RPM Package
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)